dump_to_aml.bat

```diff
@@ -52,7 +52,7 @@ md level1 level2\config level3
 
 if not exist out md out
 
-FOR %%A IN (boot recovery logo dtbo vbmeta bootloader odm oem product vendor system system_ext) DO (
+FOR %%A IN (boot recovery vendor_boot logo dtbo vbmeta bootloader odm oem product vendor system system_ext odm_ext) DO (
```

Comment:
i dont maintain windows script, not really needed.

Reply:
Fixed that and added some more code dtb related